.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": ["cozy-app"]
```

Comment:
It would be nice to have documentation on how to use that. And can we add this linter to CI (in `scripts/lint.sh`)?